src/actions/PrecursorAction.C

```diff
@@ -48,15 +48,18 @@ validParams<PrecursorAction>()
                         "random initial conditions for the precursors.");
   params.addParam<Real>("prec_scale", "The amount by which the neutron fluxes are scaled.");
   params.addParam<bool>("transient", true, "Whether to run a transient simulation.");
-  params.addParam<bool>(
+    params.addParam<bool>(
```

Comment:
Have you run `scripts/install-format-hook.sh`? This looks like a four space indent.